docs/user-documentation/creating-collections-in-islandora-7.x-2.x.md

```diff
@@ -0,0 +1,27 @@
+# Creating Collections in Islandora 7.x-2.x
```

Comment:
:+1: for what's here, but should we expand into how to recreate the functionality of Collections in Claw? Right now a Collection can have members, but there's no display associated with it. Danny has said we can do this with Views, so that's a process that will need documentation as well (and is tickets as #74 ) Maybe link this page out to that one with a brief explanation that a collection object does not come with a built-in display?

Or should the debate be over whether collection objects *should* come with some built0in display, since users will expect it?